app/scripts/components/common/datepicker/index.scss

```diff
@@ -0,0 +1,77 @@
+// @NOTE: Optimize the import paths to the file
```

Comment:
To optimize the import paths, I tried adding them in the sassrc, but for some reason the mixins were not found. I'll need to have another look in the next PRs.